cpp-unisocket/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:34
```

Comment:
Actually, all other clients are run non-Dockerized. Why is C++ client run as docker container?

Reply:
we don't have to have them run dockerized. I put is since you had it this way for the [kubernetes guides](https://github.com/hazelcast-guides/kubernetes). If not needed, we can change it.

Reply:
It's up to you, you know the C++ community the best. For other languages, we just create a sample client app and write how to run it. The difference between this guide is the [kubernetes guides](https://github.com/hazelcast-guides/kubernetes) is that:

- kubernetes guide => you run the client app in Kubernetes, so it must be Dockerized
- external kubernetes guide => you run the client app on your local machine, so it does not need to be dockerized.